packages/contracts/contracts/optimistic-ethereum/libraries/standards/IWETH9.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity =0.7.6;
+
+/// @title Interface for WETH9
+import '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+
+/// @title Interface for WETH9
```

Comment:
```suggestion
/// @title Interface for WETH9's non-ERC20 functions.  Also contains the non-ERC20 events normally present in the WETH9 implementation.
```

Reply:
Is this comment needed? It extends IERC20, so this is literally the entire interface for WETH 9

Reply:
Ah you're right, I'm just nitting on the events, how about `/// @title Interface for WETH9.  Also contains the non-ERC20 events normally present in the WETH9 implementation.`

Reply:
ok added!